<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>871</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border:none;
border-radius:10px;
background-color:transparent;
}</string>
   </property>
   <widget class="QLabel" name="ready_go_done">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: transparent;
	color: #1D1D1D;
	font: 600 25pt &quot;Epilogue&quot;;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>Ready?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cycle_counter_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: transparent;
	color: #1D1D1D;
	font: 500 15pt &quot;Epilogue&quot;;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>Number of Cycles completed: 00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="prog_lbl_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: transparent;
	color: #1D1D1D;
	font: 500 17pt &quot;Epilogue&quot;;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>Set duration of session:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    font-family: Epilogue;  
    font-size: 20pt;
    background-color: white;
	border-radius:10 pt;
    color: rgb(0,0,0);
	border:none;
}
QSpinBox:QLineEdit {
    background-color: rgb(255,255,255);
    border: none;
    padding: 4px 6px;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="prog_lbl_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: transparent;
	color: #1D1D1D;
	font: 500 17pt &quot;Epilogue&quot;;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>Breathe in, hold your breath, and breathe out for 4 seconds each</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ready_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>681</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1D1D1D;
	font: 600 25pt &quot;Epilogue&quot;;
	border-radius:30px;
	color:white;
	padding-bottom:0px
}
</string>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QLabel" name="timer_in_circle">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: none;
	color: #1D1D1D;
	font: 700 20pt &quot;Epilogue&quot;;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="AuroraBackgroundLabel" name="circle_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(
		spread: pad, 
		cx: 0.5, cy: 0.5,    
         radius: 0.8,          
         fx: 0.5, fy: 0.5,   
         stop: 0    #FFD5D5, 
         stop: 0.26 #FFB5B5, 
         stop: 1    #FFD5F6   
    );
border: 0px;
border-radius: 25px;   
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>540</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1D1D1D;
	font: 600 25pt &quot;Epilogue&quot;;
	border-radius:25px;
	color:white;
	padding-bottom:0px
}
</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1D1D1D;
	font: 600 25pt &quot;Epilogue&quot;;
	border-radius:25px;
	color:white;
	padding-bottom:0px
}
</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resume_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1D1D1D;
	font: 600 25pt &quot;Epilogue&quot;;
	border-radius:25px;
	color:white;
	padding-bottom:0px
}
</string>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#1D1D1D;
	font: 600 25pt &quot;Epilogue&quot;;
	border-radius:25px;
	color:white;
	padding-bottom:0px
}
</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="ready_go_done_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: transparent;
	color: #1D1D1D;
	font: 600 30pt &quot;Epilogue&quot;;
	border:none;
}</string>
    </property>
    <property name="text">
     <string>Meditation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <zorder>circle_label</zorder>
   <zorder>ready_go_done</zorder>
   <zorder>cycle_counter_label</zorder>
   <zorder>prog_lbl_6</zorder>
   <zorder>spinBox_2</zorder>
   <zorder>prog_lbl_5</zorder>
   <zorder>ready_btn</zorder>
   <zorder>timer_in_circle</zorder>
   <zorder>back_btn</zorder>
   <zorder>reset_btn</zorder>
   <zorder>resume_btn</zorder>
   <zorder>pause_btn</zorder>
   <zorder>ready_go_done_2</zorder>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: none;
font: 800 25px &quot;Epilogue&quot;;
color: #1D1D1D
}
</string>
   </property>
   <property name="text">
    <string>NutriFit</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>871</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#line{
	height:1px;
	background-color: #e8e8e8;
	border:none;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AuroraBackgroundLabel</class>
   <extends>QLabel</extends>
   <header>arora_bg</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>spinBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>prog_lbl_5</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>prog_lbl_6</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>ready_go_done</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>circle_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>pause_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>back_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>reset_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>cycle_counter_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ready_btn</sender>
   <signal>clicked()</signal>
   <receiver>timer_in_circle</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>resume_btn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>back_btn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>ready_btn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>ready_go_done</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>spinBox_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>prog_lbl_5</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>cycle_counter_label</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>prog_lbl_6</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>reset_btn</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>670</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>timer_in_circle</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>circle_label</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
